everviz/pages/configuration.py

```diff
@@ -0,0 +1,38 @@
+import os
+from everviz.log import get_logger
+
+
+logger = get_logger()
+
+
+def _markdown_text():
+    return """
+## The configuration file
+
+The following configuration content was used to generate the current report. 
+Everviz support the same configuration format as webviz, a basic example can be found [here](https://github.com/equinor/webviz-config/blob/master/examples/basic_example.yaml) 
+
+"""
+
+
+def _set_up_data_sources(api):
+    logger.info("Generating configuration page")
+    config_file = "everviz_webviz_config.yml"
```

Comment:
Should perhaps be a global private variable in the parent scope and passed in here so we don´t have to manually keep them in sync.